tests/conftest.py

```diff
@@ -102,6 +105,23 @@ def anon_identity():
     return identity
 
 
+@pytest.fixture(scope="module")
+def user_moderator(UserFixture, app, database, users):
+    """Admin user for requests."""
+    action_name = moderation_action.value
+    moderator = users["user_moderator"]
+
+    role = Role(name=action_name)
+    database.session.add(role)
+
+    action_role = ActionRoles.create(action=moderation_action, role=role)
+    database.session.add(action_role)
+
+    moderator.user.roles.append(role)
```

Comment:
shouldn't we just add the action role to the user and then somehow this would resolve to the correct needs? In my understanding, the action is the one that is passed to the user not the full role.... (please ignore or let's discuss IRL if I miss some basic principles on how permissions are working...)